07_Bootstrap/07_TugasPraktikum/index.html

```diff
@@ -0,0 +1,197 @@
+                    <ul class="navbar-nav">
+                        <li class="nav-item active">
+                            <a class="nav-link" href="#">Home</a>
+                        </li>
+                        <li class="nav-item active">
+                            <a class="nav-link" href="#">About</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="#">Features</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="#">Screenshot</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="#">Team</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="#">Pricing</a>
+                        </li>
+                        <li class="nav-item dropdown">
+                            <a class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" role="button" data-bs-toggle="dropdown" aria-expanded="false">
+                                Dropdown
+                            </a>
+                            <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
+                            <a class="dropdown-item" href="#">Action</a>
+                            <a class="dropdown-item" href="#">Another action</a>
+                            <a class="dropdown-item" href="#">Something else here</a>
+                            </div>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="#">Contact</a>
+                        </li>
+                    </ul>
+                </div>
+            </nav>  
+        </div> 
+    </div>
+    <div class="container content">
+        <div class="row text-center bawah">
+            <div>
+                <p style="font-size:30px">Simple Systeme Fordiscount</p>
+            </div>
+            <div class="pt-2">
+                <p style="font-size:12px">Integer Cursus Bibendum Augue Ac Cursus</p>
+            </div>
+        </div>
+        <div class="row d-flex justify-content-center pad">
+            <div class="col-3 text-center deskripsi">
+                <div class="pb-5">
+                    <img src="../07_TugasPraktikum/assets/img/svg/cloud.svg" alt="cloud">
+                </div>
+                <div class="pb-4">
+                    <h6><strong>INTRODUCTION WHATSAPP</strong></h6>
+                </div>
+                <div class="pb-2">
+                    <p>Lorem ipsum is simply dummy text of the printing and typesetting industry</p>
+                </div>
+                <div>
+                    <a href="#">Read More</a>
+                </div>                              
+            </div>
```

Comment:
kalo pengen pake symantic html untuk box image bisa pake figure > img > figcaption

reference:
[HTML `<figcaption>` Tag - w3school](https://www.w3schools.com/tags/tag_figure.asp)
[`<figcaption>`: The Figure Caption element - mdn web docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figcaption)